sched/scheduler/stateful_scheduler.go

```diff
@@ -414,9 +427,12 @@ addLoop:
 		select {
 		case newJobMsg := <-s.addJobCh:
 			receivedJob = true
-			js := newJobState(newJobMsg.job, newJobMsg.saga)
+			js := newJobState(newJobMsg.job, newJobMsg.saga, s.taskDurations)
 			s.inProgressJobs = append(s.inProgressJobs, js)
 
+			// TODO(jschiller): associate related tasks, i.e. task retries that decorate the taskId?
```

Comment:
I planned to rely on our existing metrics to see if sq latency improves and continue to manually review jobs that are taking longer than is typical. I'm open to other suggestions.